src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -164,10 +164,12 @@ export class Artwork extends React.Component<Props, any> {
 
 const styles = StyleSheet.create({
   image: {
-    marginBottom: 10,
+    marginBottom: 8,
   },
   text: {
-    fontSize: 12,
+    fontSize: 14,
+    lineHeight: 18,
```

Comment:
The text styling seen in the [designs](https://app.zeplin.io/project/5bef4c7dae2b6a3eb6a75f4a/screen/5c1ae7411253daaef2213dcb) isn't available in palette as far as I can tell, so we hardcode sizes and margins.

Reply:
`ArtworkGrids/Artwork`  isn't a new component, it's just the common grid item that gets rendered by `GenericGrid` and `InfiniteScrollGrid`. We're using it in LD and it's also used in other spots in Emission, so these styling tweaks are global.